### DISCUSSION TITLE
How To Guide: Adding Search with Meilisearch

### DISCUSSION THREAD

**Discussion by shivaylamba** [gatsbyjs/gatsby#34887] [Ideas / Feature Requests] [answered]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/gatsbyjs/gatsby/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report. Please post those things in GitHub Discussions: https://github.com/gatsbyjs/gatsby/discussions

### Summary

Add a how-to guide for integrating Meilisearch with Gatsby. 

Meilisearch has a [Gatsby plugin integration.](https://github.com/meilisearch/gatsby-plugin-meilisearch) and we have created a  how-to guide to be able to integrate Meilsearch in a Gatsby project.

**Comment by tyhopp** [comment, marked as answer]
Hi @shivaylamba, I'll close this since it's not a bug report. We can track this in the PR itself.
